src/pretix/base/models/event.py

```diff
@@ -109,6 +109,17 @@ def get_date_range_display(self, tz=None) -> str:
             return _date(self.date_from.astimezone(tz), "DATE_FORMAT")
         return daterange(self.date_from.astimezone(tz), self.date_to.astimezone(tz))
 
+    def get_date_range_display_for_organizer_page(self, tz=None) -> str:
+        """
+        Returns a formatted string containing the start date and the end date
+        of the event with respect to the current locale and to the ``show_times`` and
+        ``show_date_to`` settings taking consideration of sub-events.
+        """
+        if not self.has_subevents:
+            return self.get_date_range_display(tz)
+        tz = tz or pytz.timezone(self.settings.timezone)
+        return daterange(self.date_range_from.astimezone(tz), self.date_range_to.astimezone(tz))
```

Comment:
Where is `data_range_from` and `date_range_to` defined?

Reply:
They are defined in the query set in this file : src/pretix/presale/views/organizer.py

Reply:
Yes, it'd be great if you either document that here with a comment, or find a different way to do this than with a model method as this is very specific to one single view.